manage_breast_screening/clinics/jinja2/show.jinja

```diff
@@ -9,7 +9,7 @@
 
 {% block beforeContent %}
 {{ backLink({
-  "href": "javascript:history.back()",
+  "href": url('clinics:index'),
```

Comment:
This looks good for now, but I was wondering if this is another place where the back link should become dynamic in future, so that it remembers which filter was applied.